src/Analyzer/InterpolateNode.cpp

```diff
@@ -41,13 +44,23 @@ void InterpolateNode::updateTreeHashImpl(HashState &, CompareOptions) const
 
 QueryTreeNodePtr InterpolateNode::cloneImpl() const
 {
-    return std::make_shared<InterpolateNode>(nullptr /*expression*/, nullptr /*interpolate_expression*/);
+    auto cloned = std::make_shared<InterpolateNode>(nullptr /*expression*/, nullptr /*interpolate_expression*/);
```

Comment:
Not directly related to the change, - but why do we not clone arguments here and just passing nulls?

Reply:
They are cloned automatically by ::clone() code.